channel/types/channel_state.go

```diff
@@ -12,19 +12,44 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
```

Comment:
I would just remove the top two lines here. Everything else only has the full license header.